tests/UI/pages/BO/orders/shoppingCarts/view.js

```diff
@@ -15,11 +15,124 @@ class ViewShoppingCarts extends BOBasePage {
     super();
 
     this.pageTitle = 'View';
+
+    // Selectors
+    this.cartSubtitle = '#box-kpi-cart div.subtitle';
+    this.cartTotal = '#box-kpi-cart div.value';
+
+    // Customer Block
+    this.customerInformationBlock = '#main-div div[data-role=\'customer-information\']';
```

Comment:
```suggestion
    this.customerInformationBlock = '#main-div div[data-role="customer-information"]';
```

Reply:
No need, we have a eslint rule to use only single quotes.

Reply:
But it's not a part of the ESLint rule as these quote are inside a string, I think I agree with @Progi1984  on this review, we can avoid to escape there it's much clearer

Reply:
ok for me